src/Command/ProxySyncReleasesCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Command;
+
+use Buddy\Repman\Service\Downloader;
+use Buddy\Repman\Service\Proxy;
+use Buddy\Repman\Service\Proxy\ProxyRegister;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ProxySyncReleasesCommand extends Command
+{
+    private ProxyRegister $register;
+    private Downloader $downloader;
+
+    public function __construct(ProxyRegister $register, Downloader $downloader)
+    {
+        $this->register = $register;
+        $this->downloader = $downloader;
+
+        parent::__construct();
+    }
+
+    /**
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('repman:proxy:sync-releases')
+            ->setDescription('Sync proxy releases with packagist.org')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $proxy = $this
+            ->register
+            ->getByHost('packagist.org');
+
+        foreach ($this->feed()->channel->item as $item) {
+            list($name, $version) = explode(' ', (string) $item->guid);
+            $this->syncPackages($proxy, $name, $version);
+        }
+
+        return 0;
+    }
+
+    private function syncPackages(Proxy $proxy, string $name, string $version): void
+    {
+        foreach ($proxy->syncedPackages() as $package) {
```

Comment:
what about first dump synced packages to array with name as keys, and then use only `isset` to prevent looping again for each item